src/ExpressiveInstaller/Resources/config/routes-minimal.php

```diff
@@ -12,7 +19,7 @@
  *
  * $app->route('/contact', App\Action\ContactAction::class, ['GET', 'POST', ...], 'contact');
  */
-
-declare(strict_types=1);
-
-/** @var \Zend\Expressive\Application $app */
+return function (Application $app, MiddlewareFactory $factory, ContainerInterface $container) : void {
+    $app->get('/', App\Action\HomePageAction::class, 'home');
+    $app->get('/api/ping', App\Action\PingAction::class, 'api.ping');
```

Comment:
Shouldn't these be removed with the minimal setup?

Reply:
Yes; I must have copied more lines than I intended. Updating in a moment, with a force push.